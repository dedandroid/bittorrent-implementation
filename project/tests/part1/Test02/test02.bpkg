ident:bee4b4601ae93c62efb1278650e2c49727cdc2ddcb7f7d15cc6e3e58f3ab7329be67034cebb58ca5803bf84ece932bdda34a787542bcf5176535d7395cd77a41d8c413a55514a3c08f5568fb4c3c67d30981c2627761a222178707343609decd74e4646dccf6f19081888bdb1d4eb0c35a7bfd9b98189a91b1a3c7ed42b037382a317cc04d9d82f3397f05c4874bf73116283e97b2534477de6d321b9668d99f0b7390a52f863fd0d3d15fcf5372c11cc9f69abc932d30d03b19ae8f1f1d03adc9353e1d14a44757260c48b58d380d5c681a637871cb812b837cb32305c021c8ad200987a52365ee85b38d78238444ce2ff287a3cc6314199cd294ab73fb7ca9b9c3066d2c0401daeac7e6359211eba946d4c31f013030a1b689dbf6e07cb26f83356546677971a283c5ebbcb3965f5d2828d6f4d6d5787fb349e0693ff8e450d88ae7ecdc15484fc98a9e4cd34b89b514f0bec9adee52d1b5c4400134dbe80fc0f8eb188f6e13fd8b1cb2ef6ba43b40b389f9178759a56208fcadb0855b09bbc34bc534c8e6d49ec8a6557dac8b54617bd4008d8636bc474ee9356d1f863f7aa4ea477ddb38770c5e583c54beaed187671ae97b4a4b14762de59494232f4a6a175f0cb47f0837f54dad1313632ae85f0af06a5d956cc5112be4e0743ae12312d034a813d7f9c0afb9399adc4be7ff9cfc0942c19bb6b556e8f72c4772e6173
filename:test02.data
size:512
nhashes:3
hashes:
	caf29efc2e0394bb4a7c98ac9cac68dc7040129cee03221c810276b0f3e8beb5
	549853b248cc00cfe7588bcaa5e0cd2e20cbd99a2b48a54561eba996eed357a0
	00d5cf2ba3b6a327c963203833044af7cc2831d91e514e957cdae44de290e0c4
nchunks:4
chunks:
	f1c7a53d44db9bb1bb229fec065303314f682b243d94e9a496ddc765f28210ab,0,128
	5128b98387414567916863c65ace1b250013e5f95498cac5af78fa430dfb947f,128,128
	a5a8daaf19e0f1a14f2eaff7c4c13001538f8725cff19e43be0596fc1dca9f72,256,128
	e053eb85fe930057548361d0a1d7e758c2e0c5003739627e9d1dd9fe1145ab2d,384,128
