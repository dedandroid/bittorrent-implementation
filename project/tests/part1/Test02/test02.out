f1c7a53d44db9bb1bb229fec065303314f682b243d94e9a496ddc765f28210ab
5128b98387414567916863c65ace1b250013e5f95498cac5af78fa430dfb947f
a5a8daaf19e0f1a14f2eaff7c4c13001538f8725cff19e43be0596fc1dca9f72
e053eb85fe930057548361d0a1d7e758c2e0c5003739627e9d1dd9fe1145ab2d
